### PR TITLE
Create CLI

### PR DESCRIPTION
**Pull request by romdotdog**
There is currently `login`, `logout`, and `whoami` with a JS API for each.

### CODE REVIEW

**Review by JairusSW — APPROVED** [review]
Looks good!
Thanks!!
:)

File: aspkg-bss/assets/bootstrap/css/bootstrap.min.css [hunk 1]
```diff
@@ -3,10 +3,10090 @@
  * Copyright 2013-2021, Start Bootstrap
  * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap-creative/blob/master/LICENSE)
  */
-@import "https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800";@import "https://fonts.googleapis.com/css?family=Merriweather:400,300,300italic,400italic,700,700italic,900,900italic";
+@import "https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800";
+@import "https://fonts.googleapis.com/css?family=Merriweather:400,300,300italic,400italic,700,700italic,900,900italic";
 /*!
```
Comment (trusktr) [inline aspkg-bss/assets/bootstrap/css/bootstrap.min.css:8]: Why are we unminifying the minified files with `.min` in their name? Is this intentional?

**Comment by trusktr** [comment]
Let's take into more consideration pull request changes before merging them. We didn't need to unminify all the minified library files.

I undid the minification and formatted all code using the top-level Prettier config.

**Comment by trusktr** [comment]
Nice CLI work! We need plan somewhere exactly what we want. We can do it in an issue for example. I think all of us have different ideas, but we have agreed on things concretely.

**Pull request merged**